cmp/grid/renderers/MultiFieldRenderer.ts

```diff
@@ -96,17 +96,22 @@ function renderMainField(value, renderer, context) {
     });
 }
 
-function renderSubField({colId, label}, context) {
+function renderSubField({colId, label, labelRenderer}, context) {
     const {record, gridModel} = context,
         column = gridModel.getColumn(colId);
 
     throwIf(!column, `Subfield ${colId} not found`);
 
-    const {field, headerName, renderer} = column,
+    const {field, headerName, renderer, appData} = column,
+        {multiZoneLabelRenderer} = appData,
```

Comment:
Spoke to Lee. We want to leave appData for apps. We should provide an official column level property. Either a multizoneConfig package or a multiZoneLabelRenderer property.

Note I think it's possible apps are already expected to provide a multizone related config in appData, or that there is one attached to appData by the multizone grid model. We could consider using the "multizoneConfig" approach mentioned above for this as well.